### ISSUE TITLE
Allow changing audio latency

### ISSUE THREAD

**Issue by Zerowalker** [xenia-project/xenia#2025]
### Validation

- [X] I've read the [FAQ](https://github.com/xenia-project/xenia/wiki/FAQ).
- [X] The Xenia build used is from the master branch (not MLBS/AlexVS/Canary/pull requests, etc.)
- [X] This issue isn't for tech support (help with Xenia).
- [X] If this issue occurs in a specific game, I've done analysis to locate the faulty subsystem of the emulator and a potential reason in it.
- [X] I've checked if this issue hasn't already been reported.
- [X] My device meets the minimum requirements: https://github.com/xenia-project/xenia/wiki/Quickstart#system-requirements
- [ ] (If building) I have read the building doc: https://github.com/xenia-project/xenia/blob/master/docs/building.md

### Describe what's going wrong

The audio latency is quite high by default, i believe it's 500 ms or 1 second.
Higher is usually safer in terms of cracking etc so it might be needed sometimes.

### Describe what should happen

One should be allowed to set the latency (audio buffer size?) manually so one can achieve lower delay (by trial and error by that user).

**Comment by Rex000** [comment]
Agreed. The audio latency is just way too high and there is no way to reduce it. Even in Canary where there is a setting for it, it barely makes it a bit better. Don't know why this has been a persistent issue with Xenia for such a long time.

**Comment by Rex000** [comment]
Agreed. The audio latency is just way too high and very notable in games, and there is no way to even reduce it let alone remove it. Even in Canary where there is actually a setting to reduce it, it barely improves the latency. Don't know why this has been a persistent issue with Xenia for such a long time.
